homeassistant/components/spotify/media_player.py

```diff
@@ -56,6 +59,27 @@
     | SUPPORT_VOLUME_SET
 )
 
+TYPE_ALBUM = "album"
+TYPE_TRACK = "track"
+TYPE_ARTIST = "artist"
```

Comment:
We should standardize this. Please move this to `media_player/const.py` and call them `MEDIA_TYPE_ALBUM` etc

Reply:
After a quick chat with @jjlawren this is what we had agreed on.